styx-scheduler-service/src/main/java/com/spotify/styx/state/handlers/ExecutionDescriptionHandler.java

```diff
@@ -61,12 +61,19 @@ public void transitionInto(RunState state) {
     switch (state.state()) {
       case PREPARE:
         try {
-          final Event createdEvent =
-              Event.submit(state.workflowInstance(), getExecDescription(workflowInstance));
-          try {
-            stateManager.receive(createdEvent);
-          } catch (StateManager.IsClosed isClosed) {
-            LOG.warn("Could not send 'created' event", isClosed);
+          final Optional<Workflow> workflow = storage.workflow(workflowInstance.workflowId());
+          if (workflow.isPresent()) {
```

Comment:
No it doesn't change any existing logic, so yes still racy. This PR is about logging missing workflow as info instead of warning without trying to fix the racy problem. To fix the bigger problem, another PR? Or maybe we should just close this one as it doesn't bring much actually.